.gitignore

```diff
@@ -158,3 +158,5 @@ Output/
 
 # images
 *.png
+
+local/
```

Comment:
I don't think this is related somewhere, right?

Reply:
I put a local/ folder in the root to contain a theme for testing, then I also outputed the binary there. I can remove it. But how do you handle testing a new segment? Do you just use your jandedobbeleer theme all the time?

Reply:
I have my own config in the home folder. I adjust that one while debugging (and leave the tasks file assumed unchanged at all times).